src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -438,15 +457,17 @@ private void PublishNewPackageVersion()
             Model.RefreshCustomNodesFromDirectory(dynamoModel.CustomNodeManager, DynamoModel.IsTestMode);
             var vm = PublishPackageViewModel.FromLocalPackage(dynamoViewModel, Model, true);
             vm.IsNewVersion = true;
+            vm.IsPackageInstalled = true;
 
             dynamoViewModel.OnRequestPackagePublishDialog(vm);
         }
 
         private void PublishNewPackage()
         {
             Model.RefreshCustomNodesFromDirectory(dynamoModel.CustomNodeManager, DynamoModel.IsTestMode);
-            var vm = PublishPackageViewModel.FromLocalPackage(dynamoViewModel, Model, false);
+            var vm = PublishPackageViewModel.FromLocalPackage(dynamoViewModel, Model, true);
             vm.IsNewVersion = false;
+            vm.IsPackageInstalled = true;
```

Comment:
Will this always be true? In the case of publishing a new package, it isn't necessary that the new package will be loaded, is it?